src/amo/api/reviews.js

```diff
@@ -53,6 +53,8 @@ export type SubmitReviewParams = {|
   versionId?: number,
 |};
 
+export type SubmitReviewResponse = ExternalReviewType;
```

Comment:
I had to import it into the saga for type hinting (since Flow can't follow saga's cryptic API) so I thought it would be better to give it more specific alias.